mula/tests/integration/test_services.py

```diff
@@ -0,0 +1,194 @@
+import copy
+import time
+import unittest
+from unittest import mock
+
+from scheduler import config, models, storage
+from scheduler.connectors import services
+from scheduler.utils import remove_trailing_slash
+
+from tests.factories import PluginFactory
+
+
+class BytesTestCase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.config = config.settings.Settings()
+        self.service_bytes = services.Bytes(
+            host=remove_trailing_slash(str(self.config.host_bytes)),
+            user=self.config.host_bytes_user,
+            password=self.config.host_bytes_password,
+            source="scheduler_test",
+        )
+
+    def test_login(self):
+        self.service_bytes.login()
+
+        self.assertIsNotNone(self.service_bytes.headers)
+        self.assertIsNotNone(self.service_bytes.headers.get("Authorization"))
+
+    def test_expired_token_refresh(self):
+        self.service_bytes.get_last_run_boefje(
+            boefje_id="boefje-1",
+            input_ooi="ooi-1",
+            organization_id="org-1",
+        )
+        initial_token = copy.deepcopy(self.service_bytes.headers.get("Authorization"))
+
+        time.sleep(70)
```

Comment:
This is quite some time that gets wasted, is there any possible solution to avoid this sleep?

Reply:
Indeed, have to think about how to do this with an integration with bytes. Since bytes has a minimum token expiration of 1 minute, in order to hit that I have to wait for at least a minute

Reply:
For now, I will skip the unit test and request a feature for bytes to be able to set the token expiration to that less of a minute